src/Features/CSharp/Portable/CodeFixes/Async/CSharpAddAwaitCodeFixProvider.cs

```diff
@@ -98,7 +98,7 @@ private static bool DoesExpressionReturnTask(ExpressionSyntax expression, Semant
             }
 
             return TryGetExpressionType(expression, semanticModel, out var returnType) &&
-            semanticModel.Compilation.ClassifyConversion(taskType, returnType).Exists;
+            semanticModel.Compilation.ClassifyConversionInfo(taskType, returnType).Exists;
```

Comment:
Note: one large area where we could share lots more would be in the Cast-simplifier.  However, that code also checks things like IsBoxing/Unboxing and a bunch of other members that didn't make it to the common layer yet.  Once we figure out a story for those, we should be able to convert a lot more.